.github/workflows/packages-build-wpk.yml

```diff
@@ -0,0 +1,98 @@
+name: Packages - Build WPK packages
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag name of the Docker image to be downloaded (Use developer to set branch name. Use auto to set src/VERSION content)'
+        required: false
+        default: auto
+      linux_branch:
+        description: 'Branch name for compiling the Linux package'
+        required: false
+      windows_package:
+        description: 'Link to download package or package name in S3'
+        required: false
+      macos_package:
+        description: 'Link to download package or package name in S3'
+        required: false
+      revision:
+        description: 'Revision used to naming WPK package'
+        required: false
+        default: 1
+
```

Comment:
Let's add a `release` parameter, so that when set to `true`, it uses the current nomenclature for the release.

Reply:
Can we use this parameter to define:

"0" for development
"1" for release
Issue number for custom ID

Reply:
Done, added new parameter release_name_format to better clarify to the users.